### ISSUE TITLE
Forkgram not listed in applications to share a file to

### ISSUE THREAD

**Issue by Kabouik** [forkgram/TelegramAndroid#272]
On Android 14, if I show a photo in the Photos app for example and tap on the Share button, Forkgram is not listed there.

**Comment by Tourma** [comment]
This might be related to why I'm looking around in issues.

Have you looked under org.telegram.messenger.application loader impel
?

That's the name it comes up with in the App drawer for me for some reason.

**Comment by 23rd** [comment]
Better now?

**Comment by Kabouik** [comment]
Yes I did try to look for org.blahblah in the apps to share to but it was not there. However it's fixed now with the latest update. Thanks!
